wielder/util/terraform_credential_helper.py

```diff
@@ -79,25 +90,25 @@ def get_aws_mfa_cred_command(role_name):
 if __name__ == "__main__":
```

Comment:
move commented to test file

Reply:
fixed by moving to test. remember it is useful for cli